### ISSUE TITLE
Factor: Women's Travel Patterns - Subfactor: Location of kindergartens/childcare

### ISSUE THREAD

**Issue by timlinux** [kartoza/GEEST#39]
Source from : OSM, Humdata

````
[out:xml] [timeout:25];
 {{geocodeArea:Saint Lucia}} -> .area_0;
(
    node["amenity"="kindergarten"](area.area_0);
    way["amenity"="kindergarten"](area.area_0);
    relation["amenity"="kindergarten"](area.area_0);
);
(._;>;);
out body;
````
This factor is assessed through network analyses to calculate catchment areas using Open Route Service. After each analysis, the territory is divided into 100m x 100m rasters, and the factor score is calculated as the average score within each raster. The scoring for each type of facility is categorized by distance as follows:

0 to 400 meters: 5
401 to 800 meters: 4
801 to 1200 meters: 3
1201 to 1500 meters: 2
1501 to 2000 meters: 1
Over 2000 meters: 0

**Comment by carolinamayh** [comment]
Primary schools and kindergartens should be considered as a single feature. @osundwajeff

**Comment by ClaraIV** [comment]
linked to #40

**Comment by ClaraIV** [comment]
@osundwajeff I linked this to ticket #75, please see specifications there. I mentioned they are changed, once we reviewed it all. 
Please follow the specifications there for the renaming, sorry for any confusion.
